de/api/configuration-dev.md

```diff
@@ -23,31 +23,31 @@ Example:
 
 ```js
 module.exports = {
-  dev: (process.env.NODE_ENV !== 'production')
-}
+  dev: process.env.NODE_ENV !== 'production',
+};
 ```
 
 `server.js`
 
 ```js
-const { Nuxt, Builder } = require('nuxt')
-const app = require('express')()
-const port = process.env.PORT || 3000
+const { Nuxt, Builder } = require('nuxt');
```

Comment:
We are coming closer! However, please use semicolon-less style and only include necessary changes :relaxed: